controllers/dataprotection/backup_controller.go

```diff
@@ -389,6 +390,23 @@ func (r *BackupReconciler) patchBackupObjectMeta(
 	request *dpbackup.Request) (bool, error) {
 	targetPod := request.TargetPods[0]
 
+	encryptPassword := func() (string, error) {
+		target := request.BackupPolicy.Spec.Target
+		if target == nil || target.ConnectionCredential == nil {
+			return "", nil
+		}
+		secret := &corev1.Secret{}
+		if err := r.Client.Get(request.Ctx, types.NamespacedName{Name: target.ConnectionCredential.SecretName, Namespace: request.Namespace}, secret); err != nil {
```

Comment:
I prefer to use `client.ObjectKey` to replace `types.NamespacedName` that can avoid import of "k8s.io/apimachinery/pkg/types"